app/Http/Controllers/PostController.php

```diff
@@ -179,6 +179,28 @@ public function postVote(Request $request)
         }
     }
 
+    public function postBestAnswer(Request $request)
+    {
+        $input = $request->all();
+
+        if ($request->ajax() && !empty($input['answer_id']) && !empty($input['post_id']) && Auth::check()) {
+            $input['user_id'] = Auth::id();
+
+            if ($result = $this->postRepository->voteBestAnswer($input)) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/PostController.php</br>**Line**: 189</br>Blank line found at start of control structure